src/classes/storage.ts

```diff
@@ -61,13 +73,15 @@ export class Storage {
    * @returns {StopCluster}
    */
   public getStopClusterByGpuid(stopClusterGpuid: StopClusterGpuid): StopCluster {
-    const groundPlace: StopCluster | StopGroup = this.groundPlaces[stopClusterGpuid];
+    const placeIndex: number = this.groundPlaces.findIndex(({ gpuid }: GroundPlace) => gpuid === stopClusterGpuid);
```

Comment:
Not as optimized as before but OK 
Why not having two data structure ? 

One by key and the other as a list ? 
PS this is an opti. We will see that later when everyhting will be perfect ;)

Reply:
Thinking about it, I thought that maintaining two different structures at the same time might make CRUD methods more complex. We would have to update both structures (our array of object, and the object) each time we have to create/update or delete some place.

I prefer to lose a little perf on methods that are already fast enough (like get a specific place), and gain some on the autocomplete method :)